applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -187,10 +231,12 @@ static void stream_qos_set_cb(struct bt_audio_stream *stream)
 {
 	int ret;
 
-	ret = bt_audio_stream_enable(stream, lc3_preset_nrf5340.codec.meta,
-				     lc3_preset_nrf5340.codec.meta_count);
-	if (ret) {
-		LOG_ERR("Unable to enable stream: %d", ret);
+	if (playing_state) {
```

Comment:
Why not check the stream endpoint state?

Reply:
Because then the stream would just restart as soon as the qos cb is called, no matter if you actually want to be paused or not